src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -164,33 +160,27 @@ public GoogleBucketResource getOrCreateBucket(
         // CASE 3: we have the flight locked, but we did all of the creating.
         return createFinish(bucket, flightId, googleBucketResource);
       } else {
-        // bucket exists, but metadata record does not exist.
-        if (allowReuseExistingBuckets) {
-          // CASE 4: go ahead and reuse the bucket and its location
-          return createMetadataRecord(bucketName, projectResource, region, flightId, daysToLive);
-        } else {
-          // CASE 5:
-          throw new CorruptMetadataException(
-              "Bucket already exists, metadata out of sync with cloud state: " + bucketName);
-        }
+        // CASE 4: bucket exists
+        throw new CorruptMetadataException(
+            "Bucket already exists, metadata out of sync with cloud state: " + bucketName);
       }
     } else {
       // bucket does not exist
       if (googleBucketResource != null) {
         String lockingFlightId = googleBucketResource.getFlightId();
         if (lockingFlightId == null) {
-          // CASE 6: no bucket, but the metadata record exists unlocked
+          // CASE 5: no bucket, but the metadata record exists unlocked
```

Comment:
Thank you for iterating through all the numbers! But I think we should just get rid of all the case numbers here so modifying the file won't be so annoying in the future